producers/pairquantities.py

```diff
@@ -361,9 +368,16 @@
     name="taudecaymode_2",
     call="quantities::tau::decaymode({df}, {output}, 1, {input})",
     input=[q.dileptonpair, nanoAOD.Tau_decayMode],
-    output=[q.decaymode_2],
+    output=[q.tau_decaymode_2],
     scopes=["mt", "et", "tt"],
 )
+tau_decaymode_2_notau = Producer(
+    name="tau_decaymode_2_notau",
+    call="basefunctions::DefineQuantity({df}, {output}, -1)",
+    input=[],
+    output=[q.tau_decaymode_2],
+    scopes=["em"],
```

Comment:
`["em", "ee", "mm"]` because you also add this Producer to the `*PairQuantities` ProducerGroup.
Same comment for `tau_decaymode_1_notau`.